stable/traefik/templates/configmap.yaml

```diff
@@ -92,10 +92,32 @@ data:
     {{- end}}
     format = "{{ .Values.accessLogs.format }}"
     {{- end}}
+    {{- if and .Values.kvprovider.type .Values.kvprovider.endpoint }}
+    [{{ .Values.kvprovider.type }}]
+    endpoint = "{{ .Values.kvprovider.endpoint }}"
+    watch = {{ .Values.kvprovider.watch }}
+    prefix = {{ .Values.kvprovider.prefix }}
+    {{- if eq .Values.kvprovider.type "etcd" }}
+    useAPIV3 = true
+    {{- end }}
+    {{- if .Values.kvprovider.username }}username = {{ .Values.kvprovider.username }}{{- end }}
+    {{- if .Values.kvprovider.password }}password = {{ .Values.kvprovider.password }}{{- end }}
+      {{- if .Values.kvprovider.tls }}
+      [etcd.tls]
```

Comment:
are the TLS settings only valid for `etcd`?

Reply:
No I just missed to generalize this one